test/tests.sh

```diff
@@ -250,6 +250,44 @@ test_valid_identifiers()
         "$pipestatus"
 }
 
+test_wd_addcd_path_only()
```

Comment:
Would this test still pass if given an absolute path or a path with slashes in?

Reply:
yep! updated tests